docker/Dockerfile.ci_gpu

```diff
@@ -66,6 +66,7 @@ COPY install/ubuntu_install_redis.sh /install/ubuntu_install_redis.sh
 RUN bash /install/ubuntu_install_redis.sh
 
 # Environment variables
+ENV CUDA_HOME=/usr/local/cuda
```

Comment:
Ideally, we should not rely on CUDA_HOME, and allow some local search happening(in contrib.nvcc), which is aware of CUDA_HOME, but will also search for /usr/local/cuda by default